tools/generate_point_cloud.py

```diff
@@ -24,9 +24,8 @@ def main(args):
     parser.add_argument('--utm', 
                         help='UTM zone that AOI is located in', required=True)
     args = parser.parse_args(args)
-
-    subprocess.run(["python3", "/VisSatSatelliteStereo/stereo_pipeline.py", 
-                    "--config_file", args.config_file], check=True)
+    cmd_args = ['/bin/bash', '-c', 'source /opt/conda/etc/profile.d/conda.sh && conda activate vissat && python3 /VisSatSatelliteStereo/stereo_pipeline.py --config_file '+args.config_file]
+    subprocess.run(cmd_args, check=True)
```

Comment:
Again, not a huge deal, but keeping the lines to <80 characters would make the code a bit more readable here.